javascript/lib/api/src/model/response.ts

```diff
@@ -65,6 +65,27 @@ export class PutResponse<T> implements GenericResponse {
   }
 }
 
+export class MergeResponse<T> implements GenericResponse {
+  public constructor(
+    private readonly _value: T | null,
+    private readonly _status: number,
+    private readonly _headers: Map<string, string>
+  ) {}
+
+
```

Comment:
`wasCreated` and `wasUpdated` (like for the PutResponse) could also be added.

Reply:
Looks like putResponse en mergeResponse are the same so I removed mergeResponse and used putResponse instead.